index.html

```diff
@@ -80,7 +59,7 @@ <h2 class="center">Our Initiatives</h2>
         <div class=' col s10 offset-s1 l4'>
           <div class="card ayellow sticky-action">
             <div class="card-image waves-effect waves-block waves-light">
-              <img src="background2.jpg" class ='activator' alt="" style='height: 25vh;'>
+              <img src="background2.jpg" class ='activator' alt=""  >
             </div>
             <div class="card-content">
               <span class="card-title activator grey-text text-darken-4">NUANSA</span>
```

Comment:
It would be better to set a fixed height for the card content, so even though two cards have different length of content, they still look kinda similar.